src/stores/roles/state.ts

```diff
@@ -0,0 +1,17 @@
+import type { Role } from '@/domain/Role'
+import { type Ref, ref } from 'vue'
+
+const roles: Ref<Role[]> = ref([])
+const quizAnswer = {
```

Comment:
Don't set nothing here. Instead, define an interface inside domain folder (like Role), and use it here (like the previous line)